common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -219,6 +224,9 @@ public class CommonValue {
                         + MLModel.IS_HIDDEN_FIELD
                         + "\": {\"type\": \"boolean\"},\n"
                         + "      \""
+                        + TENANT_ID
```

Comment:
Why not add this field to model group index ?

Reply:
Yeah, I will add that when I'll work on model group apis. I just add few randomly just to show what fields are we going to add.